packages/flutter_tools/lib/src/resident_runner.dart

```diff
@@ -678,6 +678,8 @@ abstract class ResidentRunner {
   bool get isRunningRelease => debuggingOptions.buildInfo.isRelease;
   bool get supportsServiceProtocol => isRunningDebug || isRunningProfile;
 
+  Uri get uri => flutterDevices.first.devFS.baseUri;
```

Comment:
nit: document as the URI of the first connected device for mobile, and only connected device for web.

Reply:
Also when it will be null

Reply:
Done.